# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/clisp/hotfix-makemake.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

Pass --with-clisp to the module configure also when there is no config.cache.

--- clisp-2.49.20250504.gitf662209/src/makemake.in	2026-09-27 18:08:23.397540392 +0200
+++ clisp-2.49.20250504.gitf662209/src/makemake.in	2026-09-27 18:08:23.399146841 +0200
@@ -3813,7 +3813,7 @@ make_modules() {
   # is bash, config.cache may be unparsable with sh on Solaris
   echotab "      \$(SHELL) \$\$m/configure --with-clisp=\"${HEREP}/clisp -K ${LINKING_SET} ${someflags}\" --cache-file=\$\${cache} --srcdir=\$\$m \$(MODULE_CONFIGURE_FLAGS);\\"
   echotab "    else \\"
-  echotab "      \$(SHELL) \$\$m/configure --srcdir=\$\$m \$(MODULE_CONFIGURE_FLAGS); \\"
+  echotab "      \$(SHELL) \$\$m/configure --with-clisp=\"${HEREP}/clisp -K ${LINKING_SET} ${someflags}\" --srcdir=\$\$m \$(MODULE_CONFIGURE_FLAGS); \\"
   echotab "    fi ) ;\\"
   echotab "fi; \\"
   # srcdir must be correct in Makefile because clisp-link calls link.sh
